thorin-config.cmake

```diff
@@ -31,6 +31,7 @@ FIND_PATH ( THORIN_LIBS_DIR
     NAMES
         ${THORIN_OUTPUT_LIBS}
     PATHS
+        ${THORIN_LIBS_DIR}
```

Comment:
Indeed, it is not set here. But if it is set outside of the thorin-config.cmake (e.g. in a dependent project's CMakeLists.txt) and this file included via find_package(thorin) it can use the given value rather than overwrite it.